cmd/mtaBuild.go

```diff
@@ -0,0 +1,176 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"sync"
+	"text/template"
+)
+
+const templateMtaYml =
+`_schema-version: "2.0.0"
+ID: "{{.ID}}"
+version: {{.Version}}
+
+parameters:
+  hcp-deployer-version: "1.0.0"
+
+modules:
+  - name: {{.Name}}
+    type: html5
+    path: .
+    parameters:
+       version: {{.Version}}-${timestamp}
+    build-parameters:
+      builder: grunt
+      build-result: dist`
+
+
+func mtaBuild(config mtaBuildOptions, commonPipelineEnvironment *mtaBuildCommonPipelineEnvironment) error {
+	log.Entry().Info("Launching mta build")
+	return runMtaBuild(config, commonPipelineEnvironment, &command.Command{})
+}
+
+func runMtaBuild(config mtaBuildOptions, commonPipelineEnvironment *mtaBuildCommonPipelineEnvironment,
+	s shellRunner) error {
+
+	prOut, pwOut := io.Pipe()
+	prErr, pwErr := io.Pipe()
```

Comment:
handle stderr and stdout via logging framework, see https://github.com/SAP/jenkins-library/blob/master/cmd/kubernetesDeploy.go#L21

Reply:
Of course we can redirect the logging output from underlying processes through another logging framework again. Looks like some overhead and might be confusing: We might get maven logging with level `[ERROR]` decorated again with level `INFO` from the go layer. There are also edge cases where we don't get the logging directly (e.g. `echo -n "xxx"` --> suppressed newline). But that are edge cases. I have a light preference for redirecting output from processes from deeper layers directly to `stderr`.

Reply:
see above ...